.github/ISSUE_TEMPLATE/inception-request.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Inception request
+about: Suggest an idea for OFN's Inception process
+title: "[Inception Request: Name your problem statement here]"
+labels: 'Blocked:Design Input needed, Blocked: Tech Input needed, Blocked:Product
+  input neeed'
+assignees: ''
+
+---
+
+**Is your request related to a problem? Please describe in as much specific detail as possible.**
+
+*e.g. A clear and concise description of what the problem is. Ex. We want to find out how to include a process where shoppers can facilitate a better group buying process in OFN [...]*
```

Comment:
In order to reduce text:

Simply

_Ex. How can shoppers facilitate a better group buying process in OFN [...]*_

Reply:
Done!